### PR TITLE
[GLOSSARY] minor fixes to general bitcoin

### PR DESCRIPTION
**Pull request by MaxHillebrand**
this ready for review branch makes a minor rework of the glossary, just nits that I found while reading.

### CODE REVIEW

File: docs/glossary/Glossary-GeneralBitcoin.md [hunk 1]
```diff
@@ -491,15 +491,15 @@ It emphasizes the achievement of the Bitcoin protocol to establish consensus on
 ### Transaction
 
 In simple terms, a transfer of bitcoin.
-More precisely, a transaction is a signed data structure expressing a transfer of value.
+More precisely, a transaction is a signed data structure expressing a transfer of value from inputs to outputs.
```
Comment (yahiheb) [inline docs/glossary/Glossary-GeneralBitcoin.md:494]: I am not sure about this one.

Comment (MaxHillebrand) [reply docs/glossary/Glossary-GeneralBitcoin.md:494]: yeah, I was stumbling on this one too.
I just want to make clear that the main part of the tx is inputs and outputs. 

how about this

```suggestion
More precisely, a transaction is a signed data structure consisting of inputs and outputs.
```

Comment (yahiheb) [inline docs/glossary/Glossary-GeneralBitcoin.md:494]:
```suggestion
More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs
```

File: docs/glossary/Glossary-GeneralBitcoin.md [hunk 2]
```diff
@@ -491,15 +491,15 @@ It emphasizes the achievement of the Bitcoin protocol to establish consensus on
 ### Transaction
 
 In simple terms, a transfer of bitcoin.
-More precisely, a transaction is a signed data structure expressing a transfer of value.
+More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs
```
Comment (yahiheb) [inline docs/glossary/Glossary-GeneralBitcoin.md:494]:
```suggestion
More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs.
```

File: docs/glossary/Glossary-GeneralBitcoin.md (hunk 1 above)
Comment (yahiheb) [reply docs/glossary/Glossary-GeneralBitcoin.md]: I changed it to `More precisely, a transaction is a signed data structure expressing a transfer of value and consisting mainly of inputs and outputs.`

**Pull request merged**